framework/caching/DbCache.php

```diff
@@ -216,12 +219,14 @@ protected function addValue($key, $value, $duration)
         $this->gc();
 
         try {
-            $this->db->createCommand()
-                ->insert($this->cacheTable, [
-                    'id' => $key,
-                    'expire' => $duration > 0 ? $duration + time() : 0,
-                    'data' => [$value, \PDO::PARAM_LOB],
-                ])->execute();
+            $this->db->noCache(function (Connection $db) {
+                $db->createCommand()
+                    ->insert($this->cacheTable, [
+                        'id' => $key,
+                        'expire' => $duration > 0 ? $duration + time() : 0,
+                        'data' => [$value, \PDO::PARAM_LOB],
```

Comment:
I wonder that too, I guess it didn't but it also didn't hang my app anymore (see #11719).